plugins/modules/vmware_dvs_portgroup.py

```diff
@@ -624,20 +1349,37 @@ def main():
                     uplink_teaming_override=False,
                     vendor_config_override=False,
                     vlan_override=False
-                )
-            )
+                ),
+                aliases=['port_policy']
+            ),
+            netflow_enabled=dict(type='bool', default=False),
+            block_all_ports=dict(type='bool', default=False),
+            # TODO: traffic shaping
+            # TODO: traffic filtering and marking
+            # NOTE: The below parameters are deprecated starting from Ansible collection 2.0.0
+            vlan_trunk=dict(
+                type='bool',
+                default=False,
+                removed_in_version='2.0.0',
+                removed_from_collection='community.vmware',
+                ),
```

Comment:
```suggestion
            ),
```
CI breaks because of a pep8 issue:

E123: closing bracket does not match indentation of opening bracket's line